client/src/components/Chat.jsx

```diff
@@ -582,16 +582,16 @@ const Chat = () => {
                                                             }
                                                         >
                                                             Quote Reply
-                                                            <Dropdown.Item
-                                                                onClick={() => 
-                                                                    console.log(
-                                                                        'blocking'
-                                                                        )
-                                                                    }
-                                                                >
-                                                                    Block
-                                                        </Dropdown.Item>
                                                         </Dropdown.Item>
+                                                        <Dropdown.Item
+                                                             onClick={() => 
+                                                                console.log(
+                                                                    'blocking'
+                                                                    )
+                                                                }
+                                                            >
+                                                        Block
+                                                    </Dropdown.Item>
```

Comment:
looks like you are doing this directly from GitHub because it's not properly formatted

Reply:
No, i'm using VS Code but ig prettier isn't working properly. I'm checking it.